Gpumenu

ksjfhor opened the pull request:
Added changeable World Environment to Lobby and the Lab, took the one from the lobby and put it into scene management for easy reuse.
Added preset graphics options, range from potato to ultra maschine.
The presets do not set the 2D and 3D AA, for now.

So far, I uncommented the directional light on the shadow size, because it all would crash without a directional light in the main menu. 

Feel free to edit and to tell me what you think !

ksjfhor committed:
enhanced the menu

Added changeable World Environment

ksjfhor committed:
Removed my test light

now it is really okay

ksjfhor commented on COGITO/Theme/Cogito_Theme_A.tres:
Thank you for your feedback ! 
In fact, I changed nothing (at least not on purpose) on the theme.
As you may see in other PRs, this theme get updated quite often, so I remove this and try to keep this in mind :)

BrodyB replied:
Yeah, I can confirm that `Cogito_Theme_A.tres` gets file changes if you even look at it weird.

ksjfhor committed:
Delete COGITO/Theme/Cogito_Theme_A.tres

Nothing was changed, so there is no need to push.

brian-holsters commented on COGITO/Theme/Cogito_Theme_A.tres:
I think you deleted the file as a whole instead of removing the changes from the branch

ksjfhor replied:
Indeed.

ksjfhor committed:
Restore the theme

Phazorknight commented:
Just adding a comment here that I've provided some feedback and thoughts in the Issue thread here: #55

ksjfhor committed:
Further improvments

- Celling in the Lab made thicker and reduced the spotlight by the desks, moved the sunlight outside the building.
- Created a DirectionalLight group to handle the change of non existing DirectionalLight in the Scene
- Renamed a few nodes for a better and more similar reading
- Added a Headbobble option to the gameplay tab
- Commented the glow and sdfgi options plus saves and loads of the cfg

ksjfhor reviewed (commented):
Not sure about the headbobble, somehow something git ed up.

ksjfhor commented:
So the Headbobble mismatch came from PR#158..
I do not know why, but I cannot pull anything into this PR, every adjustments from now on would I have to do from hand.
sooo .... @Phazorknight what do you think now ? #55

ksjfhor committed:
see comment

-swapped out the margin for a grid in the audio tab
-added in PR#161
-More chances to apply changes :)

Phazorknight commented:
Closed due to migration to https://codeberg.org/Phazorknight/Cogito/pulls/158
